backend/substrapp/views/filters_utils.py

```diff
@@ -71,6 +71,13 @@ def _same_nature(filter_key, object_type):
     return {filter_key, object_type} <= {'algo', 'composite_algo'}
 
 
+def _model_tuple(model):
```

Comment:
Would be good to find a better name for this function: `_get_tuple_from_model()` and maybe add a small comment or a docstring?

```
# XXX This function is required in order to be able to list models using their key from the frontend.
```

It will be more explicit and would improve the code readability. Otherwise, it will be hard in my opinion to remember what's the purpose of this code.

Reply:
👍